mllib/src/test/scala/org/apache/spark/mllib/tree/DecisionTreeSuite.scala

```diff
@@ -563,18 +136,10 @@ class DecisionTreeSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(!metadata.isUnordered(featureIndex = 0))
     assert(!metadata.isUnordered(featureIndex = 1))
 
-    val (splits, bins) = DecisionTree.findSplitsBins(rdd, metadata)
```

Comment:
Seems we are losing coverage by removing the test for `findSplits` here and not adding them back in the ML test suites. Since we have other tests for `findSplits` I'm not sure if that's a problem, but if we are simply trying to _move_ all of these tests then these tests of `findSplitsBins` then they should be added in ML RandomForestTestSuite.